### PR TITLE
Save connection for multiple requests

### PR DESCRIPTION
**Pull request by private-it**

### CODE REVIEW

File: src/BW/Vkontakte.php [hunk 1]
```diff
@@ -52,6 +52,17 @@ class Vkontakte
...
+     * The type of connection
+     * @var array
+     */
+    private $persistentConnect = true;
+
+    /**
+     * The connection
+     * @var resource
+     */
+    static $connection;
```
Comment (bocharsky-bw) [inline src/BW/Vkontakte.php]: Could you change to `private static $connection;`?

File: src/BW/Vkontakte.php [hunk 2]
```diff
@@ -318,6 +332,29 @@ public function getResponceType()
...
+
+        return $this;
+    }
+
+    /**
+     * Get the status of type connection
+     *
+     * @return string
+     */
+    public function getPersistentConnect()
```
Comment (bocharsky-bw) [inline src/BW/Vkontakte.php]: I would like to use `isPersistentConnect` name for boolean types. Could you change?

File: src/BW/Vkontakte.php [hunk 3]
```diff
@@ -381,4 +427,11 @@ protected function curl($url)
 
         return $result;
     }
+
+    public function __destruct()
```
Comment (bocharsky-bw) [inline src/BW/Vkontakte.php]: Could you move this method up (below constructor)? I prefer to declare all magic methods first.

**Comment by bocharsky-bw** [comment]
This changes should increase performance a bit, I like it!

Could you also fix my comments that I pointed before I merge this PR too?

**Comment by private-it** [comment]
Why private variable has name without "_"?
Example: $_isPersistentConnect.

**Comment by bocharsky-bw** [comment]
This approach (to use `_`) used in `JS` because JS don't have access specifier (private, protected and public like in PHP). So use this in PHP is overhead and bad practice.

**Comment by bocharsky-bw** [comment]
Btw, you did more work you need :) . I wanted to rename only getter not preperty. I preffer to use issers instead of getters for boolean values. But don't worry, I fix it myself, so I merging this PR now.

Thank you @private-it again for your improvements on this project!

**Comment by private-it** [comment]
I use code style similar yii2

Example private variable:
https://github.com/yiisoft/yii2/blob/6d9fe671deb7a620f207c0a9b11f2a6355250733/framework/db/BaseActiveRecord.php#L93

Example function for set boolean value:
https://github.com/yiisoft/yii2/blob/6d9fe671deb7a620f207c0a9b11f2a6355250733/framework/db/BaseActiveRecord.php#L833

**Comment by bocharsky-bw** [comment]
Yeah, there're many different coding styles, I prefer to follow Symfony one http://symfony.com/doc/current/contributing/code/standards.html :)

Btw, there're more strong CS that we all should to follow. It's name [PSR](https://github.com/php-fig/fig-standards) and there're no any `_` for private properties/methods now.

What about issers, I think it more readable in this way. The [PSR-7](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-7-http-message.md#34-psrhttpmessagestreaminterface) already use it.
And btw, the [PropertyAccess](http://symfony.com/doc/current/components/property_access/introduction.html#using-hassers-issers) work fine with isser/hasser so I don't see any reasons do not use it :)

**Comment by private-it** [comment]
Thank you very much! =)

**Comment by bocharsky-bw** [comment]
Thank you for contribution! ;)

**Pull request merged**